tensorboard/webapp/widgets/histogram/histogram_component.ts

```diff
@@ -278,6 +278,30 @@ export class HistogramComponent implements AfterViewInit, OnChanges, OnDestroy {
     this.updateChartIfVisible();
   }
 
+  onSelectTimeRangeChange(datum: HistogramDatum) {
```

Comment:
Actually, on second thought, I find my attempt at explaining this a little too abstract. it might be clearer to say something like: "When a single step is currently selected, clicking on a histogram step will create a range incorporating the currently selected step and the clicked step. When a range is currently selected, clicking on a histogram step outside that range will expand the range to include the clicked step."